packages/lib/constants.ts

```diff
@@ -53,3 +53,4 @@ export const IS_STRIPE_ENABLED = !!(
 );
 /** Self hosted shouldn't checkout when creating teams unless required */
 export const IS_TEAM_BILLING_ENABLED = IS_STRIPE_ENABLED && (!IS_SELF_HOSTED || HOSTED_CAL_FEATURES);
+export const CALENDSO_ENCRYPTION_KEY = process.env.CALENDSO_ENCRYPTION_KEY || "";
```

Comment:
We could be easily leaking the CALENDSO_ENCRYPTION_KEY here, if there is any client side code including packages/lib/constants.ts - also this hardcodes it into the build (that's not that much of an issue and may be desirable)

Reply:
Good point

Reply:
We use this pattern in multiple places tho 
<img width="693" alt="image" src="https://user-images.githubusercontent.com/3504472/209026927-94e871f2-c3cc-497f-a2dd-164ad6dd4d92.png">

Reply:
Removed for now

Reply:
Those are OK - all those are serverside 👍  (😌)